command.go

```diff
@@ -783,7 +782,7 @@ func (c *Command) initHelpCmd() {
 			Run: func(c *Command, args []string) {
 				cmd, _, e := c.Root().Find(args)
 				if cmd == nil || e != nil {
-					c.Printf("Unknown help topic %#q.", args)
+					c.Printf("Unknown help topic %q\n", args)
```

Comment:
Yes, it works. 
How it looks like in my CLI:
![2017-02-28 10 53 41](https://cloud.githubusercontent.com/assets/13235519/23393377/39a68880-fda4-11e6-8cb0-fbce288b95c0.png)